fdbserver/TCInfo.actor.cpp

```diff
@@ -356,6 +360,18 @@ int64_t TCTeamInfo::getDataInFlightToTeam() const {
 	return dataInFlight;
 }
 
+int64_t TCTeamInfo::getLongestStorageQueueSize() const {
+	int64_t longestQueueSize = 0;
+	for (const auto& server : servers) {
+		if (server->metricsPresent()) {
+			longestQueueSize = std::max(longestQueueSize, server->getStorageQueueSize());
+		} else if (server->ssVersionTooFarBehind.get() == true) {
+			return -1;
```

Comment:
It is a little bit confusing for the return value of `-1`, won't the server be treated as `undesired` when `ssVersionTooFarBehind`?

Reply:
Is this possible that an overloaded server is failed to get a metrics? If we remove this part, it is possible that a server is overloaded but the longtestQueueSize thinks this server has 0 queue length. Do we want to add a separate one to check for this case?